### ISSUE TITLE
Contribution model minor update needed

### ISSUE THREAD

**Issue by rossfarrugia** [pharmaverse/admiral#758]
@koegerr at https://roche-gsk.github.io/admiral/articles/contribution_model.html it says "_Label each issue with a label named “user request” so we can easily identify new requests_", but I don't think external contributors will have permission to update the labels. Well I don't anyway :) i think this line can be removed.

**Comment by thomas-neitmann** [comment]
I guess it will be up to someone from the core team to properly "triage" new issues by assigning labels and the project.

**Comment by rossfarrugia** [comment]
agree @thomas-neitmann - i think good we always track new issues anyway as in the later bullet "_All created issues will be reviewed within the next sprint and the creator will receive an initial feedback via a comment._"

**Comment by rossfarrugia** [comment]
also linking in discussion here at #765 and will need to link to the new developer process page from the contribution page.

**Comment by rossfarrugia** [comment]
also need to update for @bundfussr's Slack comments:

I think there are two issue on the contribution vignette:

- The heading of the introduction is corrupted (missing space between "#" and "Introduction").
- The links to slack are pointing to the admiral_testing_01 channel. Shouldn't they refer to the general admiral channel, as admiral_testing_01 is private and to be closed?

**Comment by bms63** [comment]
Can we close this issue?

**Comment by rossfarrugia** [comment]
@bms63 don't we only close them once those devel updates get merged to main? i'm not sure of the convention. there's a whole host of these such issues that are resolved in devel code and just need to merged to main as part of the next release soon.

**Comment by bms63** [comment]
Umm...I thought we closed once we merged a PR into devel.  This would be a good distinction to make in our guides if that is not the case!

**Comment by rossfarrugia** [comment]
I would say only when merging to main personally, as GitHub automatically does this then via using linked issues: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue. Also I'd prefer to leave issues open for users to see if they haven't yet made it in to a release, as it saves people asking for the update again if they see it's already been addressed and just waiting on next release.

**Comment by thomas-neitmann** [comment]
So far we did close issues manually once they have been merged into `devel`. Otherwise they don't make it into our "Sprint Done" column on the project board. What @rossfarrugia mentioned is a good point, though, and we might want to change this approach going forward.

**Comment by bundfussr** [comment]
What about closing the issue when the PR is merged to devel but moving it to "To be Released" instead of "Sprint Done"?

Then it is clear that no more work is required for the issue and that it is not released yet.

If we keep the issue open, it is hard to see if the issue is done or if there are still open questions or work to do.

**Comment by rossfarrugia** [comment]
do users ever look at the project board though? i'd say that's more for your core dev team. also GitHub does change at the top of an issue with a merged PR to devel to say its fixed and it gives the PR link. so users do get transparency that an issue has been completed but just not yet released. 

that being said... i fully trust whatever decision you decide here, and we can then just make it clear in the contribution/development vignettes.

**Comment by bundfussr** [comment]
The column where the issue is at the project board is displayed in the project field of the issue. So you can see the state even without looking at the project board.

Should we add this topic to the agenda of the "Questions and Comments" meeting on Monday?

**Comment by bundfussr** [comment]
I have added this topic to the agenda of the "Questions and Comments" meeting on next Monday.
